File: Problem-10/PrimeNumber_InRange.py
# WAP to find a prime number between range.

import math

def isPrime(n) :
    if n <= 1:
        return False
    
    for i in range(2, int(math.sqrt(n)) + 1) :
        if n % i == 0 :
            return False
        
    return True

s = int(input("Enter start : "))
e = int(input("Enter end : "))

for i in range(s, e + 1) :
    if isPrime(i) :
        print(i, end=" ")

File: Problem-10/PrimeNumber_InRange.c
// WAP to find a prime number between range.

#include<stdio.h>
#include<stdbool.h>

bool isPrime(int n){
    if(n <= 1){
        return false;
    }

    for(int i = 2; i * i <= n; i++){
        if(n % i == 0){
            return false;
        }
    }

    return true;
}

void main(){
    int s, e;

    printf("Enter start : ");
    scanf("%d", &s);

    printf("Enter end : ");
    scanf("%d", &e);

    for(int i = s; i <= e; i++){
        if(isPrime(i)){
            printf("%d ", i);
        }
    }
}